src/main/java/com/gregtechceu/gtceu/api/machine/trait/NotifiableFluidTank.java

```diff
@@ -177,14 +224,14 @@ public boolean isLocked() {
 
     public void setLocked(boolean locked) {
         if (this.isLocked() == locked) return;
-        FluidStack fluidStack = getStorages()[0].getFluid();
+        FluidStack fluidStack = storages[0].getFluid();
```

Comment:
It was overridden in one place for the CreativeTank for some stupid logic previously, however I have since removed that. It now uses the superclass member `storages` natively. It is no longer overridden anywhere.